hasura/metadata/databases/default/tables/public_circles.yaml

```diff
@@ -120,6 +120,7 @@ select_permissions:
     - created_at
     - default_opt_in
     - deleted_at
+    - discord_webhook
```

Comment:
The column should not be exposed to the public as the discord bot could be compromised 
Currently it is only available to the circle admins via the `circle_private` view

Reply:
@zashton  thank you. I reverted this changes and came up with a much simpler solution please recheck